src/eko/anomalous_dimensions/__init__.py

```diff
@@ -94,37 +95,39 @@ def gamma_ns(order, mode, n, nf):
 
     See Also
     --------
-        eko.anomalous_dimensions.lo.gamma_ns_0 : :math:`\gamma_{ns}^{(0)}(N)`
-        eko.anomalous_dimensions.nlo.gamma_nsp_1 : :math:`\gamma_{ns,+}^{(1)}(N)`
-        eko.anomalous_dimensions.nlo.gamma_nsm_1 : :math:`\gamma_{ns,-}^{(1)}(N)`
-        eko.anomalous_dimensions.nnlo.gamma_nsp_2 : :math:`\gamma_{ns,+}^{(2)}(N)`
-        eko.anomalous_dimensions.nnlo.gamma_nsm_2 : :math:`\gamma_{ns,-}^{(2)}(N)`
-        eko.anomalous_dimensions.nnlo.gamma_nsv_2 : :math:`\gamma_{ns,v}^{(2)}(N)`
+        eko.anomalous_dimensions.as1.gamma_ns : :math:`\gamma_{ns}^{(0)}(N)`
+        eko.anomalous_dimensions.as2.gamma_nsp : :math:`\gamma_{ns,+}^{(1)}(N)`
+        eko.anomalous_dimensions.as2.gamma_nsm : :math:`\gamma_{ns,-}^{(1)}(N)`
+        eko.anomalous_dimensions.as3.gamma_nsp : :math:`\gamma_{ns,+}^{(2)}(N)`
+        eko.anomalous_dimensions.as3.gamma_nsm : :math:`\gamma_{ns,-}^{(2)}(N)`
+        eko.anomalous_dimensions.as3.gamma_nsv : :math:`\gamma_{ns,v}^{(2)}(N)`
     """
     # cache the s-es
     sx = np.full(1, harmonics.harmonic_S1(n))
     # now combine
     gamma_ns = np.zeros(order + 1, np.complex_)
-    gamma_ns[0] = lo.gamma_ns_0(n, sx[0])
+    gamma_ns[0] = as1.gamma_ns(n, sx[0])
     # NLO and beyond
     if order >= 1:
         # TODO: pass the necessary harmonics to nlo gammas
-        if mode == "p":
-            gamma_ns_1 = nlo.gamma_nsp_1(n, nf)
+        if mode == 10101:
+            gamma_ns_1 = as2.gamma_nsp(n, nf)
         # To fill the full valence vector in NNLO we need to add gamma_ns^1 explicitly here
-        elif mode in ["m", "v"]:
-            gamma_ns_1 = nlo.gamma_nsm_1(n, nf)
+        elif mode in [10201, 10200]:
+            gamma_ns_1 = as2.gamma_nsm(n, nf)
+        else:
+            raise NotImplementedError("Non-singlet sector is not implemented")
```

Comment:
```suggestion
            raise NotImplementedError(f"Non-singlet sector {mode} is not implemented")
```

Reply:
It looks like f-strings are not yet supported by numba, but you can do:

```suggestion
            raise NotImplementedError("Non-singlet sector " + str(mode) + " is not implemented")
```